docs/user-documentation/content_types.md

```diff
@@ -0,0 +1,126 @@
+
+## Modify a Content Type
+
+If you have deployed your Islandora 8 with the Islandora Demo configuration, you will already have a Repository Item content type available, with pre-configured fields. In the Admin menu, go to Structure >> Content Types and find the Repository Item content type. Select *Manage Fields*.
+
+![a screenshot of the Add Content Type page](/docs/assets/islandora8_managefields.png)
+
+You will see a list of the fields that are already available in this content type. 
+
+### Add a Field
+
+Let’s add a new field where a user can indicate if the repository item needs to be reviewed:
+
+1. Click “Add Field”
+1. Since this is a “yes/no” decision, we will use “Boolean” and give the field a name. [List of Drupal 8 FieldTypes, FieldWidgets, and FieldFormatters](https://www.drupal.org/docs/8/api/entity-api/fieldtypes-fieldwidgets-and-fieldformatters)
+1. Now we configure how the field is stored in the Drupal database. For this field type we can select how many values will be allowed. Set this value to "1":
+1. Now we configure how the field is described (including its display label and the help text for when it appears on a form) and constraints on its use. In this case, the field will be required for this Content Type, and will be set to “on” by default: 
+
+![a screenshot of the field settings page](/docs/assets/islandora8_fieldsettings.png)
+
+*Save*. Now we have added our new field: 
+
+![a screenshot of a "Needs Review?" field in the Drupal field UI](/docs/assets/islandora8_newfield.png)
+
+And it appears in the ingest form when we try to create a new repository object:
+
+![a screenshot of a "Needs Review?" field appearing at the bottom of a new node form](/docs/assets/islandora8_newfieldinform.png)
+
+!!! tip New fields, with the exception of Typed Relation fields, are not automatically indexed in Fedora and the triple-store. Update the Content Type's RDF Mapping [link that section of the docs] to enable indexing the field. 
+
+!!! tip To add new behavior based on the results of this new field, check out [link to Context docs]
+
+### Change the Form Display
+
+Now let’s move our new field to a different part of the form. In the Admin menu, return to Structure > Content Types and find the Repository Item content type again. Select Manage form display.
+
+1. All of the fields in this content type are available, in a list, with a simple drag-and-place UI. Drag the *Needs Review?* Field to the top of the form. We can also change the way the Boolean options are displayed, with radio buttons as opposed to a single checkbox. Different display options will be available depending on field type. For more information, please check out [List of Drupal 8 FieldTypes, FieldWidgets, and FieldFormatters](https://www.drupal.org/docs/8/api/entity-api/fieldtypes-fieldwidgets-and-fieldformatters)
```

Comment:
suggest change Needs Review? to **new field**

Reply:
Change: `Different display options will be available depending on field type`
To: Different display options will be available **from the dropdown menu** depending on field type.